src/spec-common/injectHeadless.ts

```diff
@@ -512,38 +512,41 @@ async function runLifecycleCommand({ lifecycleHook }: ResolverParameters, contai
 					name: progressName,
 					status: 'succeeded',
 				});
+			} catch (err) {
+				infoOutput.event({
+					type: 'progress',
+					name: progressName,
+					status: 'failed',
+				});
+				if (err?.cmdOutput) {
```

Comment:
Only write the output when `name` is set. Otherwise the output was already written (see `printMode` above).

Reply:
Done